### DISCUSSION TITLE
What would be the reason of spaces bucket File Listing to be out of sync with the UI

### DISCUSSION THREAD

**Discussion by yordis** [digitalocean/terraform-provider-digitalocean#847] [Q&A] [answered]
Hey peep, I recently setup a basic project, here is the full version of it without the variables and providers

```hcl
resource "digitalocean_project" "shared_project" {
  name        = "shared"
  description = "Shared is used for infrastructure and data that is shared amongst all the applications"
  purpose     = "Operational / Developer tooling"
  environment = "Production"
}

resource "digitalocean_spaces_bucket" "terraform" {
  name   = "${var.prefix}-tf"
  region = "nyc3"
  acl    = "private"
}
```

Noticed that I put `acl = "private"` which I was expected to be "Restrict File Listing". Is that correct?

But when I went to the UI, the "Restrict File Listing" option wasn't selected, I had to then click on it and save it to become blue as I showed below, but before that was gray as if I didn't select it.

![Screen Shot 2022-07-13 at 1 42 11 AM](https://user-images.githubusercontent.com/4237280/178659301-c5d0a9e5-1491-4dee-8e24-9f63cf00060c.png)

Is `acl="private"` the same as `Restrict File Listing` option? If so, why would be the reason that wasn't selected correctly? Otherwise, what configuration I am looking to set up in order to accomplish the `Restrict File Listing` option?

Thanks in advance.

**Comment by danaelhe** [comment, marked as answer]
Hi there!

Thank you so much for this write up. You are correct, this is not expected UI behavior. I have brought this up to our internal storage engineering team and they were aware of the bug and are currently implementing a fix for it.

**Reply by yordis** [reply]
Much love for y'all ❤️

**Comment by andrewsomething** [comment]
@yordis This was a UI issue where it was not correctly reflecting the current setting. It should be resolved now. Let us know if you're still running into any issues.
